packages/suite/src/components/wallet/CoinjoinSummary/CoinjoinStatus.tsx

```diff
@@ -143,20 +156,24 @@ export const CoinjoinStatus = ({
     const menuRef = useRef<HTMLUListElement & { close: () => void }>(null);
     const theme = useTheme();
 
-    const { maxRounds, signedRounds, paused, phaseDeadline, sessionDeadline, phase } = session;
+    const { maxRounds, signedRounds, paused, phaseDeadline, sessionDeadline, phase, interrupted } =
+        session;
 
     const progress = calculateSessionProgress(signedRounds, maxRounds);
     const isPaused = !!paused;
 
+    const isInterrupted = !!interrupted;
```

Comment:
I think logic here could rely on the Tor status, no?

Reply:
Interrupted is a property of the CoinJoin account, it actually does not have to be related to Tor.

Reply:
👍 like this https://github.com/trezor/trezor-suite/issues/6610